quarkchain/cluster/miner.py

```diff
@@ -263,6 +233,8 @@ def _mine_new_block_async(self):
         # 5 sec interval magic number
         if not self.current_work or now - self.current_work.header.create_time > 5:
             block = await self.create_block_async_func()
+            if not block:
+                raise RuntimeError("Created empty block")
```

Comment:
the message is confusing. shouldn't it be "Failed to create block"?

Reply:
will do